packages/mockserver-config-writer/src/app-info.ts

```diff
@@ -23,13 +34,17 @@ export function getMainServiceDataSource(manifest: Manifest): ManifestNamespace.
  * Get the data sources of type 'OData' from manifest.
  *
  * @param manifest - the parsed manifest.json
+ * @param dataSourceType - optional to specify the type of OData sources to filter
  * @returns data sources of type 'OData'
  */
-export function getODataSources(manifest: Manifest): { [k: string]: ManifestNamespace.DataSource } {
+export function getODataSources(
+    manifest: Manifest,
+    dataSourceType = ODataSourceType.OData
```

Comment:
If we use existing type, this would look like
```suggestion
    dataSourceType: ManifestNamespace.DataSourceEnum['type'] = 'OData'
```